cli/tests/environment_validate.sh

```
@@ -0,0 +1,51 @@
+echo "Testing environment validate command..."
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)